doc/user_doc/vic_installation/vch_installer_examples.md

```diff
@@ -361,22 +361,21 @@ This example deploys a VCH with the following configuration:
 
 - Specifies the image store, cluster, bridge network, and name for the VCH.
 - Specifies <i>vsphere_admin</i> in the `--target` option, to identify the user account with which to deploy the VCH.
-- Specifies <i>vsphere_user</i> and its password in the `--ops-user` and `--ops-password` options, to identify the user account with which the VCH runs.
+- Specifies <i>vsphere_user</i> and its password in the `--ops-user` and `--ops-password` options, to identify the user account with which the VCH runs. The user account that you specify in `--ops-user` must exist before you deploy the VCH. For information about the permissions that the `--ops-user` account requires, see [Set Up the `--ops-user` User Account](set_up_ops_user.md).
```

Comment:
Expanded: 

"The user account that you specify in `--ops-user` must  is different to the vSphere Administrator account that you use for deployment, and must exist before you deploy the VCH. "